src/common/is-null-or-undefined.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export function isNullOrUndefined(value: any): boolean {
```

Comment:
If you make the return type a [type predicate](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) instead of just `boolean`, you can avoid needing to use `results!` on L66 of `store-data-to-scan-node-result-converter.ts`.

```suggestion
export function isNullOrUndefined(value: any): value is null | undefined {
```

Reply:
As discussed, this method has gone away